README.md

```diff
@@ -11,38 +11,52 @@ Put `wsl2-ssh-agent` binary in your favorite directory in WSL2, for example, `$H
 ```
 curl -L -O https://github.com/mame/wsl2-ssh-agent/releases/latest/download/wsl2-ssh-agent
 ```
+
 If you are under ARM64 architecture, download the `arm64` binary instead:
+
 ```
 curl -L -O https://github.com/mame/wsl2-ssh-agent/releases/latest/download/wsl2-ssh-agent-arm64
 ```
+
 Change permisions so the binary is executable:
+
 ```
 chmod 755 wsl2-ssh-agent
 ```
 
 **If you are using ArchLinux, you can install the [wsl2-ssh-agent](https://aur.archlinux.org/packages/wsl2-ssh-agent) package from the AUR (maintained by @Hill-98).**
 
-### 2. Modify `.bashrc` (or `.zshrc` if you are using `zsh`)
+### 2. Modify your shell's rc file
 
+Bash or Zsh
 Add the following line to `.bashrc` (or `.zshrc` if you are using `zsh`).
 
 ```
 eval $($HOME/wsl2-ssh-agent)
 ```
 
+Fish
+Add the following lines to config.fish
+
+```
+if status is-login
+  $HOME/wsl2-ssh-agent | source
```

Comment:
Shouldn't it be `$HOME/wsl2-ssh-agent --shell fish | source` ?

Reply:
Yup, good catch. Will fix.